zkevm-circuits/src/evm_circuit/execution/extcodesize.rs

```diff
@@ -25,10 +25,9 @@ pub(crate) struct ExtcodesizeGadget<F> {
     reversion_info: ReversionInfo<F>,
     tx_id: Cell<F>,
     is_warm: Cell<F>,
-    exists: Cell<F>,
     code_hash: Cell<F>,
-    code_size: Cell<F>,
-    code_size_bytes: [Cell<F>; 8],
+    not_exists: IsZeroGadget<F>,
+    code_size: RandomLinearCombination<F, 8>,
```

Comment:
8 -> N_BYTES_U64

Reply:
Tanks!  Applied in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1046/commits/8c84bc31ed1ffc74c31508b5818e90d8762dd450